xyz.parlatype.Parlatype.json

```diff
@@ -0,0 +1,76 @@
+{
+    "app-id": "xyz.parlatype.Parlatype",
+
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+
+    "command": "parlatype",
+
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=home:ro",
+        "--talk-name=org.freedesktop.secrets",
```

Comment:
Why is this required? Does it support the secrets portal?

Reply:
Removed in https://github.com/flathub/flathub/pull/5312/commits/0fc1e2a948614cea5466b7bcb939a08b08d44fa1